src/flamenco/runtime/tests/fd_exec_instr_test.c

```diff
@@ -483,27 +486,30 @@ _context_create( fd_exec_instr_test_runner_t *        runner,
   ctx->txn_ctx   = txn_ctx;
   ctx->funk_txn  = funk_txn;
   ctx->acc_mgr   = acc_mgr;
-  ctx->valloc    = fd_scratch_virtual();
+  ctx->valloc    = fd_alloc_virtual( alloc );
```

Comment:
The instruction context should still use the scratch allocator, but its parent contexts (slot/txn) should use the workspace allocator.

This matches the setup of the ledger tool.